cmd/amp/stack-ps.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"github.com/appcelerator/amp/api/client"
+	"github.com/appcelerator/amp/api/rpc/stack"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+// StackPs command to list stack tasks
+var StackPs = &cobra.Command{
+	Use:   "ps [OPTIONS] STACK",
+	Short: "List the tasks in the stack",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return stackPs(AMP, cmd, args)
+	},
+}
+
+func init() {
+	flags := StackPs.Flags()
+	flags.Bool("no-trunc", false, "Do not truncate output")
+	flags.Bool("no-resolve", false, "Do not map IDs to Names")
+	flags.StringP("filter", "f", "", "Filter output based on conditions provided")
+	StackCmd.AddCommand(StackPs)
+}
+
+func stackPs(amp *client.AMP, cmd *cobra.Command, args []string) error {
+	manager := newManager(cmd.Flag("verbose").Value.String())
+	ctx, err := amp.GetAuthorizedContext()
+	if err != nil {
+		manager.fatalf("%v\n", err)
+	}
+	if err := amp.Connect(); err != nil {
+		manager.fatalf("Amp server is not available\n")
+	}
+	if len(args) < 1 {
+		manager.fatalf("Need stack name as first argument")
```

Comment:
first comment:
If we launch a stack (ampcore) here, when the stack is already running then docker try to update it, It's the regular behavior.
Since the image we used for amp is "local" and so not in the dockerhub registry, we'll got this messages. It's not critical.
second comment:
yes, it misses a "\n" in the message string